src/GraphQL.SystemTextJson/ROMJsonConverter.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GraphQLParser;
+
+namespace GraphQL.SystemTextJson
+{
+    /// <summary>
+    /// A custom JsonConverter for reading or writing a <see cref="ROM"/> object.
+    /// </summary>
+    public class ROMJsonConverter : JsonConverter<ROM>
+    {
+        /// <inheritdoc/>
+        public override void Write(Utf8JsonWriter writer, ROM value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value);
```

Comment:
Direct writes ROM into `Utf8JsonWriter`, no string allocations.

Reply:
Just curious - where would this be useful?  Where are ROMs returned within an ExecutionResult?

Reply:
Path. Names of fields that parsed from original query.

Reply:
Seems so.